README.md

```diff
@@ -1,2 +1,217 @@
+
+### View - Reactive Way
+
+Stay aware to think in a way to declare all the user's actions and screen's actions related as `Observable`.
+
+```kotlin
+interface LoginView : View<LoginState> {
+  fun doLoginClicks(): Observable<Unit>
+  fun activityResults(): Observable<Pair<Int, Intent>>
+}
+```
+
+### RxStore
+> `RxStore` is an implementation of `Store` that works with Reactive Extensions
+
+To create one you just need to pass an `State` class as parameter and define the related initial state:
+
+```kotlin
+RxStore(LoginState(StateType.Initial))
+```
+
+The stream created by it can be handled inside the `View` implementation.
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_login)
+
+    store.stateChanges()
+         .observeOn(AndroidSchedulers.mainThread())
+         .subscribe { bind(it) }
+}
+
+override fun bind(newState: LoginState) {
+   when (newState.type) {
+     StateType.Success -> success()
+     StateType.Error -> error()
+     StateType.Loading -> loading()
+   }
+}
+```
+
+
+### RxJob
+> `RxJob` is an implementation of `Job` that works with Reactive Extensions
+
+
+You just need to inherit your use case class from `RxJob` (specifying the type of the input parameter) and override the `bind(input: T):Completable` method with the properly work.
+
+>You must handle all the possible edge cases into your job. You should not allow your job to throw an exception back to the `Hub`.
+
+As it is returns a `Completable` you should handle the possible returns on the `RxStore` subscription, with `doOnSubscribe`, `doOnSuccess` and `doOnError` as shown in the next example:
+
+```kotlin
+class DoGoogleLoginJob @Inject constructor(
+    private val store: RxStore<LoginState>,
+    private val firebase: GoogleFirebaseAuth) : RxJob<Intent>() {
+
+  override fun bind(input: Intent): Completable {
+    return firebase.firebaseAuthWith(intent = input)
+        .doOnSubscribe { store.update { setType(StateType.Loading) } }
+        .doOnSuccess { store.update { setType(StateType.Success) } }
+        .doOnError { store.update { setType(StateType.Error) } }
+        .toCompletable()
+        .onErrorComplete()
+  }
+}
+```
+
+> In this example we are injecting the local parameters `store` and `firebase` with [dagger](https://github.com/google/dagger) we strong recommend that.
+
+### RxHub
+> `RxHub` is an implementation of `Hub` that works with Reactive Extensions
+
+You connect the `View` and the `RxJob`'s layers through the operator `bind(job: RxJob<T>)`(that's why all of your `View`'s methods needs to return an `Observable`).
+
+After that just need to inherit the class `RxHub<T>`(specifying the `View` it will connect with) and override the `connect(view: T)` method with the binds to `RxJobs` you need.
+
+We have already implemented the `disconnect()` method to release resources by disposing all the bound jobs.
```

Comment:
Suggestion:

It contains multiple `bind()` extensions functions and every `RxJob` result (`Completable`) is automatically added on `CompositeDisposable`.

The `RxHub` class overrides the `disconnect()` function and call `CompositeDisposable.clear()`.